docs/api/python/io.md

```diff
@@ -14,54 +14,8 @@ data including
     mxnet.image
 ```
 
-It will also show how to write an iterator for a new data format.
-
-A data iterator reads data batch by batch.
-
-```python
->>> data = mx.nd.ones((100,10))
->>> nd_iter = mx.io.NDArrayIter(data, batch_size=25)
->>> for batch in nd_iter:
-...     print(batch.data)
-[<NDArray 25x10 @cpu(0)>]
-[<NDArray 25x10 @cpu(0)>]
-[<NDArray 25x10 @cpu(0)>]
-[<NDArray 25x10 @cpu(0)>]
-```
-
-If `nd_iter.reset()` is called, then reads the data again from beginning.
-
-In addition, an iterator provides information about the batch, including the
-shapes and name.
-
-```python
->>> nd_iter = mx.io.NDArrayIter(data={'data':mx.nd.ones((100,10))},
-...                             label={'softmax_label':mx.nd.ones((100,))},
-...                             batch_size=25)
->>> print(nd_iter.provide_data)
-[DataDesc[data,(25, 10L),<type 'numpy.float32'>,NCHW]]
->>> print(nd_iter.provide_label)
-[DataDesc[softmax_label,(25,),<type 'numpy.float32'>,NCHW]]
-```
-
-So this iterator can be used to train a symbol whose input data variable has
-name `data` and input label variable has name `softmax_label`.
-
-
-```python
->>> data = mx.sym.Variable('data')
->>> label = mx.sym.Variable('softmax_label')
->>> fullc = mx.sym.FullyConnected(data=data, num_hidden=1)
->>> loss = mx.sym.SoftmaxOutput(data=data, label=label)
->>> mod = mx.mod.Module(loss)
->>> print(mod.data_names)
-['data']
->>> print(mod.label_names)
-['softmax_label']
->>> mod.bind(data_shapes=nd_iter.provide_data, label_shapes=nd_iter.provide_label)
-```
-
-Then we can call `mod.fit(nd_iter, num_epoch=2)` to train `loss` by 2 epochs.
+A detailed tutorial is available at
```

Comment:
i prefer to have a short description here so that readers don't need to click the link. we can mention that they can get more details through a link.